wiiu/vino/vino.ts

```diff
@@ -387,74 +474,74 @@ export default interface VinoAPI {
+	 * @param left - Left position in screen
+	 * @param top -  Top position in screen
+	 * @param width - Width of loading icon
+  	 * @param height - Height of loading icon
 	 */
-	loading_setIconRect: (x1: number, x2: number, y1: number, y2: number) => void;
+	loading_setIconRect: (left: number, top: number, width: number, height: number) => void;
 
 	/**
-	 * Opens the browser to the given URL
+	 * Opens the browser to the given URL, (if browser is closed, returns to TVii)
 	 * @param url - URL to jump to
-	 * @param unk - Unknown
+	 * @param confirm - Show confirmation message if true, jump inmediately if false
 	 */
-	jumpToBrowser: (url: string, unk: boolean) => void;
+	jumpToBrowser: (url: string, confirm: boolean) => void;
 
 	/**
 	 * Opens the eShop
-	 * @param unk1 - Unknown. Title ID?
-	 * @param unk2 - Unknown
+	 * @param query - eShop query (version=1.0.0&scene=detail&dst_title_id=ID_HERE&src_title_id=ID_HERE)
+	 * @param confirm - Show confirmation message if true, jump inmediately if false
 	 */
-	jumpToEShop: (unk1: string, unk2: boolean) => void;
+	jumpToEShop: (query: string, confirm: boolean) => void;
 
 	/**
 	 * Opens Miiverse
-	 * @param unk - Unknown
+	 * @param confirm - Show confirmation message if true, jump inmediately if false
 	 */
-	jumpToMiiverse: (unk: boolean) => void;
+	jumpToMiiverse: (confirm: boolean) => void;
 
 	/**
 	 * Opens Miiverse to a specific post
 	 * @param postID - Post ID
-	 * @param unk - Unknown
+	 * @param confirm - Show confirmation message if true, jump inmediately if false
 	 */
-	jumpToMiiversePostId: (postID: string, unk: boolean) => void;
+	jumpToMiiversePostId: (postID: string, confirm: boolean) => void;
 
 	/**
-	 * Unknown
-	 * @param unk1 - Unknown
-	 * @param unk2 - Unknown
-	 * @param unk3 - Unknown
+	 * Opens an video on demand app with a URL parameter
+	 * @param titleID - Titles title ID (hex)
+	 * @param URL - URL parameter to send to the VOD app
+	 * @param confirm - Show confirmation message if true, jump inmediately if false
 	 */
-	jumpToVod: (unk1: string, unk2: string, unk3: boolean) => void;
+	jumpToVod: (titleID: string, URL: string, confirm: boolean) => void;
 
 	/**
 	 * Open an installed title
 	 * @param titleID - Titles title ID (hex)
-	 * @param unk - Unknown
+	 * @param confirm - Show confirmation message if true, jump inmediately if false
 	 */
-	jumpToTitle: (titleID: string, unk: boolean) => void;
+	jumpToTitle: (titleID: string, confirm: boolean) => void;
 
 	/**
-	 * Unknown
-	 * @param unk - Unknown
+	 * Opens System Settings on the TV Remote screen
+	 * @param confirm - Show confirmation message if true, jump inmediately if false
 	 */
-	jumpToSettingsTvRemote: (unk: boolean) => void;
+	jumpToSettingsTvRemote: (confirm: boolean) => void;
 
 	/**
-	 * Closes the app gracefully
+	 * Closes the app transitioning to the Wii U Menu/suspended app
 	 */
 	exit: () => void;
 
 	/**
-	 * Force closes the app
+	 * Closes the app directly to the HBM
```

Comment:
This is a bad description. Why would this open the homebrew menu? There's no way that's intended usage